src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -52,7 +52,8 @@ def create(self, username, password):
                 mgr.ACCESS_CTRL_DB.reset_attempt(username)
                 mgr.ACCESS_CTRL_DB.save()
                 token = JwtManager.gen_token(username)
-                token = token.decode('utf-8')
+                # For backward-compatibility with PyJWT versions < 2.0.0
```

Comment:
Please specify which version is returning bytes and which str.

Reply:
Done.